src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -53,49 +51,22 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
-        try {
-            const clientConfig: ClientConfig = await getClientConfig(this.parent, this.databaseName);
-            const children: AzExtTreeItem[] = [
-                new PostgresFunctionsTreeItem(this, clientConfig),
-                new PostgresTablesTreeItem(this, clientConfig)
-            ];
-
-            if (this.parent.supportsStoredProcedures()) {
-                children.push(new PostgresStoredProceduresTreeItem(this, clientConfig));
-            }
-
-            return children;
-        } catch (error) {
-            const parsedError: IParsedError = parseError(error);
-
-            if (parsedError.errorType === invalidCredentialsErrorType) {
```

Comment:
How come all of this stuff is gone? This kind of stuff is worth keeping, even if it only applies for the Azure case

Reply:
I just checked that it was gone. I was working with enabling authentication for attached accounts  and testing somethings, might have not undone the changes here.